google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/BigQueryRpc.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.bigquery.spi;
+package com.google.cloud.bigquery.spi.v2;
```

Comment:
Do we need to move the factories as well? If not, will `BigQueryRpcFactory` always construct the latest major version? And then, how would we create older versions? Or would it only be possible to construct a single major version with the factory, as the manual layer is tied to a specific one.